modules/price/src/TwigExtension/PriceTwigExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\commerce_price\TwigExtension;
+
+use Drupal\commerce_price\Price;
+use Drupal\Core\Template\TwigExtension;
```

Comment:
There's no need to extend the core TwigExtension, extend the Twig one instead (\TwigExtension).